programs/issue-2-temp-calculator/run.py

```diff
@@ -0,0 +1,39 @@
+def celsius_to_fahrenheit(celsius):
+    fahrenheit = celsius * 9/5 + 32
+    print("{}°C to {}°F".format(celsius, fahrenheit))
+
+
+def fahrenheit_to_celsius(fahrenheit):
+    celsius = (fahrenheit - 32) * 5/9
+    print("{}°F to {}°C".format(fahrenheit, celsius))
+
+
+def menu():
+    print("Program do konwersji temperatury w różnych skalach. Jaką konwersje chcesz wykonać?")
+    print("1) Stopnie Celsjusza na Fahrenheita")
+    print("2) Stopnie Fahrenheita na Celsjusza")
+    while True:
+        try:
+            user_options = int(input("Wybierz: "))
```

Comment:
czepiając się, powinno być w sumie chyba raczej "user option", w liczbie pojedynczej. W końcu użytkownik wybiera jedną opcje ;) Ale mało ważne. Możesz poprawić to przy okazji innego Pull Requesta